tests/jballerina-integration-test/src/test/resources/configurables/testPathProject/Config.toml

```diff
@@ -0,0 +1,21 @@
+[configPkg]
```

Comment:
do we have any tests where config.toml in tests directory?

Reply:
yes. 
https://github.com/ballerina-platform/ballerina-lang/blob/1be4fb25c48bd3c7ee7f9e81ac76f83ca292bc59/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java#L72